core/src/trezor/lvglui/scrs/template.py

```
@@ -364,34 +371,40 @@ def show_address(self, evm_chain_id: int | None = None):
+            self.derive_btn.add_style(
+                StyleWrapper().bg_color(lv_colors.ONEKEY_GRAY_3),
+                0,
+            )
 
-        # address
-        self.group_address = ContainerFlexCol(
-            self.content_area, self.derive_btn, pos=(0, 8), padding_row=0
-        )
+            # address
+            self.group_address = ContainerFlexCol(
+                self.content_area, self.derive_btn, pos=(0, 8), padding_row=0
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)